seirmo/models.py

```diff
@@ -1,7 +1,13 @@
+"""#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#"""
 #
```

Comment:
Could you possible remove the comment `# Forward model` it's not really necessary. The license header is enough, i.e.

```
#
# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
# which is released under the BSD 3-clause license. See accompanying LICENSE.md
# for copyright notice and full license details.
# 
```
Could you also please remove the """ at the beginning and end of the header?